File: common/src/main/java/net/impleri/itemskills/integrations/kubejs/events/EventsBinding.java
package net.impleri.itemskills.integrations.kubejs.events;

import dev.latvian.mods.kubejs.event.EventGroup;
import dev.latvian.mods.kubejs.event.EventHandler;

public abstract class EventsBinding {
    public static final EventGroup GROUP = EventGroup.of("ItemSkillEvents");

    public static final EventHandler RESTRICTIONS = GROUP.server("register", () -> RestrictionsRegistrationEventJS.class);
}

File: common/src/main/kotlin/net/impleri/itemskills/integrations/kubejs/events/EventsBinding.kt
package net.impleri.itemskills.integrations.kubejs.events

import dev.latvian.mods.kubejs.event.EventGroup
import dev.latvian.mods.kubejs.event.EventHandler

object EventsBinding {
  val GROUP: EventGroup = EventGroup.of("ItemSkillEvents")
  val RESTRICTIONS: EventHandler = GROUP.server("register") { RestrictionsRegistrationEventJS::class.java }
}
